truckersmp_cli/args.py

```diff
@@ -89,6 +89,18 @@ def check_args_errors():
             logging.info("Unable to find logged in steam user automatically.")
             sys.exit("Need the steam account name (-n name) to update.")
 
+    # check for Wine desktop size
+    if Args.wine_desktop:
+        split_size = Args.wine_desktop.split("x")
+        if len(split_size) != 2:
+            sys.exit("Desktop size ({}) must be 'WIDTHxHEIGHT' format".format(
+                Args.wine_desktop))
+        try:
+            if int(split_size[0]) <= 0 or int(split_size[1]) <= 0:
```

Comment:
Shouldn't we enforce something sensible here like 720p?
A 1x1 desktop seems a bit useless…